conf/ds-image-behaviour.yaml

```diff
@@ -27,7 +27,16 @@ fallback: ${nonexisting.property:-87}
 licensemodule:
   url: http://localhost:10001/ds-license/v1/
 
+
 # Default images returned if there is no access to image or image(id) does not exist
 images:
   no_access: no_access.jpg
-  non_existing: non_existing.jpg
\ No newline at end of file
+  non_existing: non_existing.jpg
+
+# Image sizes to determine thumbnail from full size. Licence module determine access to images.
+# 'Thumbnails' and 'Fullsize' are two different presentation types. 
+# If custom IIIF or IIP parameters except for size are defined it will always be fullsize. If only size are defined and smaller than these limit, then it will be thumbnail  
+thumbnail:
+  maxHeight: 150
+  maxWidth: 150
```

Comment:
Using the '-" af seperator now. Can see that what is mostly used in ds-present and ds-discover.
But there is also use of all lowercase such as 'stoponerrors'. I prefer the - version for readability.

Reply:
Latest commit changed the key for the license module from `licensemodule` to `license-module`. This makes it inconsistent with the other ds-projects as well as the JavaDoc for the `DsLicenseClient` which mentions `licensemodule`. Please change it back to `licensemodule`. Adjusting "all" keys to confirm to a common schema is a separate and non-trivial task (DISC-371).

Separate from this, the **only** place I know of where `-` (dash) is used as a separator in out DS-projects is in ds-present as the single key `ds-storage`. On the other hand, `_` (underscore) is used quite a lot throughout the projects. Maybe you are thinking of that character? So please don't use `-` (dash) as separator.

Reply:
Changed back. Another example of "-": JSON-LD  (in ds-present)

Reply:
The dash in JSON-LD is dictated by the enum in the OpenAPi specification:
```
        - name: format
          in: query
          description: |
            The delivery format for the record:
            * JSON-LD: [Linked Data in JSON](https://json-ld.org/) (default)
            * MODS: [Metadata Object Description Schema](http://www.loc.gov/standards/mods/)
            * SolrJSON: [Solr JSON Formatted Index Updates](https://solr.apache.org/guide/8_8/uploading-data-with-index-handlers.html#json-formatted-index-updates)
            * raw: Metadata unchanged from the source.
          schema:
            type: string
            enum:  ['JSON-LD', 'MODS', 'SolrJSON', "raw"]
            example: 'JSON-LD'
            default: 'JSON-LD'
```

Avoiding it might be worth doing. This would either restrict the possible enum values (problematic from a user perspective) or require a change to the YAML structure parsing, so that the enum value is matched to a YAML value instead of a YAML key.

Reply:
`licensemodule` was changed back, but `no-access`, `non-existing`, `max-height` and `max-width`still uses dash. Is that on purpose?